sites/docs/src/content/docs/tutorials/migrate_to_topics/update_pipelines.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Update pipelines to use topic channels
+description: How to update pipelines to use the new topic channels feature
+---
+
+Adapting pipelines to use topic channels for version outputs is a straightforward process explained in the steps below:
+
+1. Update the template of the pipeline to the latest version that includes support for topic channels. This is supported starting from version 3.5.0.
+
+1. Pull the latest changes made to modules using `nf-core modules update`.
+
+1. Look for issues in the pipeline code where `versions` is an invalid output of a process. Remove these outputs since these are now handled by the topic channels.
```

Comment:
this point needs to be expanded and maybe get an example error message.

Also a guide on how to handle new modules when your pipeline is using an old template would be great, because not everyone will do a template update before updating the modules

Reply:
because quite high level modules incl. MultiQC were already transformed, I think documentation on how to use them is needed with the release. 
We released the last template three weeks ago and have an conversion rate of 30% in dev and 8% released branches: https://nf-co.re/pipeline_versions So a lot of people will run into this scenario.